apps/voting/contracts/Voting.sol

```diff
@@ -251,6 +271,7 @@ contract Voting is IForwarder, AragonApp {
         vote_.metadata = _metadata;
         vote_.snapshotBlock = getBlockNumber() - 1; // avoid double voting in this very block
         vote_.totalVoters = token.totalSupplyAt(vote_.snapshotBlock);
+        vote_.supportRequiredPct = supportRequiredPct;
```

Comment:
🚨 The cached `vote_.supportRequiredPct` is currently not being used to check whether a vote can be executed in `canExecute()`.

https://github.com/aragon/aragon-apps/blob/97ca6b0ca0d07795208b7901903de470fa75035f/apps/voting/contracts/Voting.sol#L210

Reply:
Good catch, thanks!! I see that the test for it below is wrong too.